mlcomp/parallelm/extra/sagemaker/monitor/job_monitor_base.py

```diff
@@ -96,9 +224,18 @@ def _job_status(self, describe_response):
         pass
 
     @abc.abstractmethod
-    def _report_online_metrics(self, describe_response):
+    def _report_extended_online_metrics(self, describe_response):
         pass
 
     @abc.abstractmethod
-    def _report_final_metrics(self, describe_response):
+    def _report_extended_final_metrics(self, describe_response):
+        pass
```

Comment:
Why this class doesn't inherit from ABC to make it abstract? (or __metaclass__ = ABCMeta)

Reply:
Fixed